docker-compose.yml

```diff
@@ -19,3 +19,17 @@ services:
       tty: true
       volumes:
         - '.:/sdk_codegen'
+
+    typescript:
+      environment:
+        - LOOKERSDK_BASE_URL=${LOOKERSDK_BASE_URL}
+        - LOOKERSDK_API_VERSION=${LOOKERSDK_API_VERSION}
```

Comment:
This shouldn't be required any more. We should resolve that

Reply:
I tried to remove API_VERSION and more tests failed and complained it was missing. Not warnings.